<launch>
    <param name="pose_stamp_source_topic" value="/mavros/local_position/pose" />
    <param name="pose_stamp_target_topic" value="" />

    <node pkg="px4_mavros_controller" type="pose_transfer" name="pose_transfer_test" output="screen" >
    </node>
</launch>